reposerver/repository/repository.go

```diff
@@ -28,6 +28,8 @@ import (
 	"github.com/argoproj/argo-cd/util/ksonnet"
 	"github.com/argoproj/argo-cd/util/kube"
 	"github.com/argoproj/argo-cd/util/kustomize"
+
+	"golang.org/x/sync/semaphore"
```

Comment:
Please check if this is available in standard library `sync/semaphor`

Reply:
unfortunately it is not available